### PR TITLE
Feature/815 duplicate label

### PR DESCRIPTION
**Pull request by koenedaele**

### CODE REVIEW

**Comment by koenedaele** [comment]
@Wim-De-Clercq Can you take a look at why the tests are failing? The results are different on my own machine and sometimes vary between runs (4 tests failing on 3.8, 3 on 3.9 and 3.10, 5 on my local machine running 3.10), so I assume it's some kind of race condition or threading issue or I'm not initializing it all correctly.

**Comment by Wim-De-Clercq** [comment]
For me the test DB is wrong because sqlite does not support altering tables a whole lot. So our alembic revision of concept_id INT -> STR does not work.

That's why for me, and travis, the tests are failing with `'1' != 1` because the provider you import has INT IDs, and we expect them to be strings after importing. 

This is a correct assumption, but the test DB is still configured with INT IDs so they never become strings.

I can only assume that your own database you use for testing is either postgres, or is a manually edited sqlite with STR concept.concept_id.

**Comment by Wim-De-Clercq** [comment]
The correct alembic would be
```py
# revision identifiers, used by Alembic.
revision = '88214d86a9d1'
down_revision = '79ff53a30228'


def upgrade():
    with op.batch_alter_table('concept') as batch_op:
        batch_op.alter_column('concept_id', type_=sa.String)


def downgrade():
    with op.batch_alter_table('concept') as batch_op:
        batch_op.alter_column('concept_id', type_=sa.Integer)
```
Then a whole lot of other issues pop up though. Like `get_next_cid` is missing a cast to int:
```py
    def get_next_cid(self, conceptscheme_id, id_generation_strategy):
        if id_generation_strategy == IDGenerationStrategy.NUMERIC:
            max_id = self.session.execute(
               select(func.max(cast(Thing.concept_id, Integer)))  # over here
               .filter_by(conceptscheme_id=conceptscheme_id)
            ).scalar_one()
            return max_id + 1 if max_id else 1
        elif id_generation_strategy == IDGenerationStrategy.GUID:
            return str(uuid.uuid4())
        else:
            raise ValueError("unsupported id_generation_strategy")
```
And more...

**Comment by koenedaele** [comment]
But in the develop branch with skosprovider_sqlalchemy 2.0.1 everything is fine (same settings, everything default, so no manual edits). The change from skosprovider_sqlalchemy 2.0.1 to 2.10 does not seem like it should be causing these kinds of failing tests?

**Comment by Wim-De-Clercq** [comment]
Yes, but all the tests which are failing in this PR are tests which also have changes from this PR, that is why develop is fine.

But this does not mean that these changes are wrong. I do believe they are correct and it's in fact the code which is wrong.
On develop both the code AND the tests are wrong and hide the issues because they cancel themselves out.

**Comment by koenedaele** [comment]
So, what do we do? First fix the test against the current develop branch and then rebase this PR on the updated develop branch? Or fix the tests in this branch (but fix all of them)?

I personally prefer the first solution since that keeps everything nicely separated. Your thoughts?

File: atramhasis/scripts/import_file.py [hunk 1]
```diff
@@ -262,10 +238,9 @@ def main(argv=sys.argv):
     if args.cs_uri:
         cs_uri = args.cs_uri
         cs_label = args.cs_label if args.cs_label else input_name.capitalize()
-        cs = create_conceptscheme(cs_label, cs_uri)
-    else:
-        cs = create_conceptscheme_from_skos(provider.concept_scheme)
-    provider_to_db(provider, cs, session)
+        args.concept_scheme = create_conceptscheme(cs_uri, cs_label)
+    provider = file_to_provider_function(**vars(args))
```
Comment (Wim-De-Clercq) [inline atramhasis/scripts/import_file.py:242]: There's also a `file_to_provider_function` call on line 237. I assume that one should be deleted.

**Review by Wim-De-Clercq — CHANGES_REQUESTED** [review]

**Review by Wim-De-Clercq — APPROVED** [review]

**Pull request merged**